cranelift-wasm/src/module_translator.rs

```diff
@@ -127,7 +127,6 @@ pub fn translate_module<'data>(
         let code = section.get_code_section_reader()?;
         parse_code_section(code, environ)?;
 
-        reader.skip_custom_sections()?;
```

Comment:
While adding the test cases below, noticed that this line causes us to skip right past the name section if there isn't a data section in the binary.